app/forms/form.rb

```diff
@@ -2,6 +2,7 @@ class Form
   include ActiveModel::Model
   include ActiveModel::Attributes
   include ActiveModel::Serialization
+  include ActiveModel::Validations::Callbacks
```

Comment:
Nice, once this is in master I'll rebase my pr to use this rather than using define model call backs
https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/c32857dc81622e2338cbe382a94487c386c78357/app/forms/form.rb#L5